pom.xml

```diff
@@ -137,6 +138,37 @@
       </plugins>
     </pluginManagement>
     <plugins>
+      <plugin>
+        <groupId>net.ltgt.gwt.maven</groupId>
+        <artifactId>gwt-maven-plugin</artifactId>
+        <configuration>
+          <moduleName>org.optaplanner.workbench.OptaPlannerWorkbench</moduleName>
+          <style>PRETTY</style>
+          <logLevel>INFO</logLevel>
+          <!-- compile goal configuration, see https://tbroyer.github.io/gwt-maven-plugin/compile-mojo.html -->
+          <compilerArgs>
+            <arg>-strict</arg>
+            <arg>-saveSource</arg>
+            <arg>-saveSourceOutput</arg>
+            <arg>extra</arg>
+          </compilerArgs>
+          <!-- codeserver goal configuration, see https://tbroyer.github.io/gwt-maven-plugin/codeserver-mojo.html -->
+          <codeserverArgs>
+            <arg>-strict</arg>
+          </codeserverArgs>
+          <!-- GWT Super Dev Mode: point codeserver to backend to overwrite *.nocache.js -->
+          <launcherDir>optaplanner-wb-webapp/target/optaplanner-wb-webapp</launcherDir>
+          <systemProperties>
+            <!-- Make IDE war run config and GWT Super Dev Mode use same
+              deployment context to avoid duplicate, conflicting deployment -->
+            <errai.dev.context>optaplanner-wb</errai.dev.context>
```

Comment:
That's a property for the EmbeddedWildFlyLauncher. It configures the servlet context when deploying in super dev mode in case you don't want to use the root context.